db/warehouse/migrate/20240628180603_index_import_tables.rb

```diff
@@ -0,0 +1,30 @@
+class IndexImportTables < ActiveRecord::Migration[7.0]
+  disable_ddl_transaction!
```

Comment:
@eanders we aren't using async (concurrent indexes). Probably want this in a ddl tx?

Reply:
So, I've had terrible deployment experiences with that recently.  (And couldn't get it working correctly locally either.) I'm not sure if it's our deployment infrastructure or what, but It tends to throw errors that it can't get a lock unless we add indexes synchronously.  Maybe we need to mess with the lock timeouts?

Reply:
Oh yeah, I just meant that there's no reason to run this outside of a ddl tx right? So we can remove this line
```suggestion
```

Reply:
Yes! that was probably left over from trying to follow the strong migrations guidance.